pymesos/process.py

```diff
@@ -147,7 +147,12 @@ def abort(self):
 
     def stop(self):
         Process.abort(self)
-        self.join()
+        assert (threading.current_thread() != self.accept_t), "Can't call stop in accept_t thread."
+        logger.debug("join ioloop thread.")
+        self.accept_t.join()
+        if threading.current_thread() != self.delay_t:
```

Comment:
because I always calls `driver.run()` in main thread and in some special case when I call `driver.start()` to run the scheduler, then a `join` is needed. I think ensuring depay_t finished is the user's responsibility not the pymesos's responsibility. the library just provides the feature, how to use it's the user's work